digiroad2-geo/src/main/scala/fi/liikennevirasto/digiroad2/GeometryUtils.scala

```diff
@@ -302,11 +302,24 @@ object GeometryUtils {
     }
   }
 
-  def calculateBearing(geom: Seq[Point]): Int = {
-    //TODO Test bearing calculation
+  def calculateBearing(geom: Seq[Point], assetMValue: Option[Double] = None, geomLength: Option[Double] = None): Int = {
```

Comment:
Since we are expecting to have both **assetMValue and geomLength** at same time
I suggest
def calculateBearing(geom: Seq[Point], pointMValueAndRoadLinkLength Option[(Double, Double)] = None)
or 
def calculateBearing(geomInfo: (Seq[Point], Length) , pointMValue Option[Double] = None)
if it possible